src/crafting.cpp

```diff
@@ -863,8 +863,19 @@ const recipe *select_crafting_recipe( int &batch_size )
             redraw = true;
             keepline = true;
         } else if (action == "FILTER") {
+          std::string desc = std::string("Special prefixes:\n") +
+            "  [t] search tools\n" +
+            "  [c] search components\n" +
+            "  [q] search qualities\n" +
+            "  [s] search skills\n" +
+            "  [S] search skill used only\n" +
+            "Examples:\n" +
+            "  t:hammer\n" +
+            "  c:two by four\n" +
+            "  q:butchering\n" +
+            "  s:cooking";
             filterstring = string_input_popup(_("Search:"), 85, filterstring,
-                                              _("Search tools or component using prefix t. \nSearch skills using prefix s, or S for skill used only. \n (i.e. \"t:hammer\" or \"c:two by four\" or \"s:cooking\".)"));
+                                              _(desc.c_str()));
```

Comment:
OK, this'll need reworked before we can merge, see BevapDin's on translation protocol.